src/SprykerMiddleware/Zed/Process/Business/Stream/JsonStream.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Business\Stream;
+
+// @codingStandardsIgnoreFile
+class JsonStream
+{
+    /**
+     * @var resource
+     */
+    protected $subhandle;
+
+    /**
+     * @var array
+     */
+    protected $data = [];
+
+    /**
+     * @var string
+     */
+    protected $mode = '';
+
+    /**
+     * @var int
+     */
+    protected $position;
+
+    /**
+     * @return bool
+     */
+    public function stream_close()
+    {
+        return fclose($this->subhandle);
+    }
+
+    /**
+     * @return bool
+     */
+    public function stream_eof()
+    {
+        return $this->position >= count($this->data);
+    }
+
+    /**
+     * @return bool
+     */
+    public function stream_flush()
+    {
+        fwrite($this->subhandle, json_encode($this->data));
+        return fflush($this->subhandle);
+    }
+
+    /**
+     * @param string $data
+     *
+     * @return void
+     */
+    public function stream_write($data)
+    {
+        $this->data[] = json_decode($data);
+    }
+
+    /**
+     * @return bool|string
+     */
+    public function stream_read()
+    {
+        if (!$this->stream_eof()) {
+            return json_encode($this->data[$this->position++]) . PHP_EOL;
+        }
+
+        return false;
+    }
+
+    /**
+     * @return array
+     */
+    public function stream_stat()
+    {
+        return fstat($this->subhandle);
+    }
+
+    /**
+     * @param string $path
+     * @param string $mode
+     * @param int $options
+     * @param string $opened_path
+     *
+     * @return bool
+     */
+    public function stream_open($path, $mode, $options, &$opened_path)
+    {
+        $parts = parse_url($path);
+        if ($opened_path === null) {
+            $this->subhandle = fopen($parts['host'] . ':' . $parts['path'], $mode, $options);
```

Comment:
third parameter for fopen() is use_include_path which is boolean, looks like it's not correct to mix it up with options flags